src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java

```diff
@@ -199,9 +212,40 @@ public static TimeOperator2 from(TimeOperator to) {
     }
 
     /**
+     * Enumeration for geometry types
+     */
+    enum GeometryOperand {
+
+        Envelope,
+        Point,
+        LineString,
+        Polygon,
+        ArcByCenterPoint,
+        CircleByCenterPoint,
+        Arc,
+        Circle,
+        ArcByBulge,
+        Bezier,
+        Clothoid,
+        CubicSpline,
+        Geodesic,
+        OffsetCurve,
+        Triangle,
+        PolyhedralSurface,
+        TriangulatedSurface,
+        Tin,
+        Solid;
+
+        public static String asString(GeometryOperand go) {
+            return "gml:" + go.name();
+        }
```

Comment:
Because in the capabilities I need ``<ogc:GeometryOperand>gml:Envelope</ogc...>``. But now I see that putting the namespace as a constant here is really bad.

Alternative: Put the appending of the namespace in the handler. Or would you @autermann say that this is also "stupid enough" to be put in the encoder?